reference/reference-styling.html

```diff
@@ -249,9 +254,16 @@ <h2>Layers</h2>
 </div>
         </div>
 
+        <h2>CSS Variables</h2>
+        <p>These CSS variables can be set in a CSS rule on a puzzle.</p>
```

Comment:
nit: 

These CSS variables can set in a CSS rule on a puzzle. The CSS rules may also be [modified through JavaScript](https://www.w3schools.com/css/css3_variables_javascript.asp).

Reply:
Sure why not :-)